src/test/osd/TestOSDMap.cc

```diff
@@ -426,7 +426,7 @@ class OSDMapTest : public testing::Test,
         cout << "<<<<<PGs distribution:" << std::endl;
       }
       int high_load_after = 0;
-      for (auto i = 0 ; i < weights.size() ; i++) {
+      for (auto i = 0 ; std::cmp_less(i, weights.size()) ; i++) {
```

Comment:
`std::cmp_less` is a bit confusing due to its novelty. Wouldn't `auto i = 0U` or `size_t i = 0` be enough?
Anyway, this is so minor that I'm not going to block the PR.